User-flow PR for test

AmitManna99 opened the pull request.

Abhijit2505 commented:
![Capture eduAlgo Doc](https://user-images.githubusercontent.com/52751963/102583589-fd5b8f00-412a-11eb-8a40-9c12bf0bd1eb.PNG)

@AmitManna99 , We don't want this part to be removed, I have tested your code and I can't see this part. Please look into it. Also don't remove the left side bar that contains the outline.

Before removing any feature or portion from the existing code, let us know first.

AmitManna99 commented:
@Abhijit2505 , Actually I did remove any left side bar. I just modified its look. Also, I putted sidenav code into a new file which will be retrieve by  JS. It will reduce the repetition of code.

I also putted a right side bar.

**1. Left Sidebar for Website Outline
2. Right Sidebar for In-Page Outline**

![Screenshot 2020-12-18 150637](https://user-images.githubusercontent.com/54736237/102600603-0c4f3b00-4145-11eb-8b43-2439159d1b19.png)


I just removed the section you mentioned in the picture because people can directly go to the page through the left side nav. (_Instead of clicking on the nav which scroll the page, then click again on the item. That increases the number of click._) Although I can bring the section back if you want.

Abhijit2505 commented:
Great, but these parts of the page were not visible in my local server, are
there any specific dependencies that I should know about?

On Fri, Dec 18, 2020 at 3:31 PM Amit Manna <notifications@github.com> wrote:

> @Abhijit2505 <https://github.com/Abhijit2505> , Actually I did remove any
> left side bar. I just modified its look. Also, I putted sidenav code into a
> new file which will be retrieve by JS. It will reduce the repetition of
> code.
>
> I also putted a right side bar.
>
>
> *1. Left Sidebar for Website Outline 2. Right Sidebar for In-Page Outline*
>
> [image: Screenshot 2020-12-18 150637]
> <https://user-images.githubusercontent.com/54736237/102600603-0c4f3b00-4145-11eb-8b43-2439159d1b19.png>
>
> I just removed the section you mentioned in the picture because people can
> directly go to the page through the left side nav. (*Instead of clicking
> on the nav which scroll the page, then click again on the item. That
> increases the number of click.*) Although I can bring the section back if
> you want.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eduAlgo/documentation/pull/7#issuecomment-747997255>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMSO4W7XYAAU45AH3Y7F47DSVMSBBANCNFSM4U4THX4A>
> .
>


-- 
<https://about.me/abhijittripathy_25?promo=email_sig&utm_source=product&utm_medium=email_sig&utm_campaign=edit_panel&utm_content=thumb>
Abhijit Tripathy
https://abhijit2505.github.io/portfolio-site/

AmitManna99 commented:
I moved this section to a new file **navbar.html** and created a separated **sidenav.css** file file.

This is the file structure I have in my local server linked to each other.

![Screenshot 2020-12-18 165005](https://user-images.githubusercontent.com/54736237/102609332-47576b80-4151-11eb-953e-d0bfca646ec2.jpg)

Abhijit2505 commented:
![Capture bb](https://user-images.githubusercontent.com/52751963/102734519-04baac80-4366-11eb-9674-9ba5ec053fcc.PNG)


I have cloned your repo and hosted it in my local server, the right side is visible but the left side rendered by JS is not. See the website is hosted with GitHub pages, so is there any more dependencies I should know about?

I am taking two of my web developers in loop @amoghpete @himanshu-0624 for this issue.
